canonical/juju-dashboard
# WD-5543- Combine panel css

## Done

- Remove duplicate panel css.
- Combine features in the Panel component.

## QA

- Check that each panel works as it did before (there will be some small visual changes but the functionality should be unchanged).
- Check that the panels look OK at mobile and desktop sizes.

## Details

https://warthogs.atlassian.net/browse/WD-5543
Fixes: https://github.com/canonical/juju-dashboard/issues/1526.
Fixes: https://github.com/canonical/juju-dashboard/issues/1523.
